langserver/ast.go

```diff
@@ -59,7 +59,7 @@ func goRangesToLSPLocations(fset *token.FileSet, nodes []*ast.Ident) []lsp.Locat
 
 func goRangeToLSPLocation(fset *token.FileSet, pos token.Pos, end token.Pos) lsp.Location {
 	return lsp.Location{
-		URI:   "file://" + fset.Position(pos).Filename,
```

Comment:
`path2uri` implementations typically need to add a leading slash if the path matches a windows path. That is typically done by testing against a drive letter followed by a colon as the first path segment.